samples/bluetooth/central_uart/README.rst

```diff
@@ -15,6 +15,36 @@ On Nordic Semiconductor's development kits, the UART 1 peripheral is typically g
 
 Any data sent from the Bluetooth LE unit is sent out of the UART 1 peripheral's TX pin.
 
+Debugging
+*********
+
+Debug messages are printed by RTT logger. You will be able to display RTT console
```

Comment:
I would add a line to explain that they are printed on RTT in this specific sample because the console is used to send/read data.

Reply:
Ok added